src/Entity/BlockInteractor.php

```diff
@@ -14,10 +14,10 @@
 namespace Sonata\PageBundle\Entity;
 
 use Doctrine\ORM\EntityManager;
+use Doctrine\Persistence\ManagerRegistry;
```

Comment:
We need the doctrine persistence dependency for this

Reply:
Of course! Thank you very much! 👍